deploy/crds/noobaa.io_bucketclasses_crd.yaml

```diff
@@ -8,6 +8,10 @@ spec:
     description: Placement
     name: Placement
     type: string
+  - JSONPath: .spec.namespacePolicy
```

Comment:
rebase and regenerate the CRDs. it should generate it as `apiextensions.k8s.io/v1` now

```
make gen-api
```

Reply:
fixed